release/mixer.sh

```diff
@@ -20,9 +20,11 @@ SCRIPT_DIR=$(dirname "$(realpath "${BASH_SOURCE[0]}")")
 var_load_all
 
 fetch_bundles() {
-    log_line "Fetching downstream bundles:"
+    log_line "Fetching bundles:"
+    local bundles_dir="${WORK_DIR}/bundles"
+    git clone --quiet "${BUNDLES_REPO}" "${bundles_dir}"
     rm -rf ./local-bundles
-    git clone --quiet "${BUNDLES_REPO}" local-bundles
+    mv "${bundles_dir}/${BUNDLES_REPO_SRC_DIR}" ./local-bundles
```

Comment:
Better to formalize that the default is an empty string? Maybe add in the top of the script:
`BUNDLES_REPO_SRC_DIR=${BUNDLES_REPO_SRC_DIR:-""}` or something near to this.

Reply:
On a second thought, this needs to be documented on the stub for the config repo. And there it will be obvious that a undefined (or empty) is the default.